linkerd/app/outbound/src/http/server.rs

```diff
@@ -157,6 +160,36 @@ impl errors::HttpRescue<Error> for ServerRescue {
             return Ok(errors::SyntheticHttpResponse::not_found(error));
         }
 
+        if errors::is_caused_by::<policy::HttpInvalidPolicy>(&*error) {
+            return Ok(errors::SyntheticHttpResponse::internal_error(
+                error.to_string(),
+            ));
+        }
+        if errors::is_caused_by::<policy::HttpRouteInvalidRedirect>(&*error) {
+            tracing::warn!(%error);
```

Comment:
i notice we warn on this error...do we want to warn on other invalid policy errors such as `HttpInvalidPolicy`?